src/retype/__init__.py

```diff
@@ -495,10 +495,12 @@ def convert_annotation(ann):
 
 def normalize_node(node):
     """Normalizes nodes to match pytree.convert and flatten power nodes"""
-    if len(node.children) == 1:
-        return node.children[0]
 
-    node.children = [normalize_node(i) for i in node.children]
+    if isinstance(node, Node):
```

Comment:
I probably should have done this in #30, but since it won't touch the children it doesn't have to touch